example/src/main/java/com/thefuntasty/mvvmsample/ui/form/FormViewModel.kt

```diff
@@ -1,13 +1,26 @@
 package com.thefuntasty.mvvmsample.ui.form
 
-import com.thefuntasty.mvvm.BaseViewModel
+import com.thefuntasty.mvvm.crinteractors.BaseCrViewModel
+import com.thefuntasty.mvvmsample.domain.GetFormFlowInteractor
+import com.thefuntasty.mvvmsample.domain.SaveFormInteractor
 import javax.inject.Inject
 
-class FormViewModel @Inject constructor() : BaseViewModel<FormViewState>() {
+class FormViewModel @Inject constructor(
+    private val saveFormInteractor: SaveFormInteractor,
+    private val getFormFlowInteractor: GetFormFlowInteractor
+) : BaseCrViewModel<FormViewState>() {
 
     override val viewState = FormViewState("", "")
 
+    override fun onStart() {
+        getFormFlowInteractor.execute({
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/2f66916fc6a03ec33808803d0586fd2c6ef9556b/example/src/main/java/com/thefuntasty/mvvmsample/ui/form/FormViewModel.kt#L16"></span>Parentheses around the lambda ({<br />
            viewState.storedContent.value = “${it.first} ${it.second}”<br />
        }) are unnecessary and can be removed.</td>
    </tr>
  </tbody>
</table>